addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -588,7 +588,7 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
         $ribbons.removeClass(htmlClasses);
 
         $ribbons.addClass(ribbon.html_class || '');
-        $ribbons.attr('style', `background-color: ${ribbon.bg_color || ''} !important`);
+        $ribbons.attr('style', `background-color: ${ribbon.bg_color || 'inherit'}`);
```

Comment:
I think you should keep `!important` here and make it work and remove related code from [python side](https://github.com/odoo/odoo/commit/9ee115b58342b3e0dbc11081e7ff752c10f8bfa9). No? :thinking: